README.md

```diff
@@ -39,7 +39,7 @@ __Options__
 
 `opts` is an object specifying the fields and their corresponding validation rules.
 
-* Each key is a field name in the post data (e.g. 'name', 'user.name') with optional search scopes: `query`, `body` and `params`. Field name and scopes are separated by `:`. If no scope is specified, all scopes are searched.
+* Each key is a field name in the post data (e.g. 'name', 'user.name') with optional search scopes: `header`, `headers`, `query`, `body` and `params`. Field name and scopes are separated by `:`. If no scope is specified, all scopes are searched.
```

Comment:
Maybe make it clear that header and headers are equivalent. Like: "with optional search scopes: `header` (or `headers`), `query`...